### PR TITLE
 Cozy UI Transpiled

### PR DESCRIPTION
**Pull request by ptbrowne**
Use transpiled version of cozy-ui
 Upgrade cozy-authentication (remove the bug where aria-hidden=true elements
 are not displayed)

### CODE REVIEW

File: src/ducks/transactions/TransactionModal.spec.jsx [hunk 1]
```diff
@@ -9,7 +9,7 @@ import pretty from 'pretty'
 import getClient from 'test/client'
 import pick from 'lodash/pick'
 
-jest.mock('preact-portal', () => ({ children }) => children)
+// jest.mock('preact-portal', () => ({ children }) => children)
```
Comment (drazik) [inline src/ducks/transactions/TransactionModal.spec.jsx]: Is it normal ?

Comment (kosssi) [inline src/ducks/transactions/TransactionModal.spec.jsx]: We can remove it?

File: src/components/SharingIcon/SharingIcon.styl [hunk 2]
```diff
@@ -3,7 +3,7 @@ purple=#a75bcb
 green=#35ce68
 
 .sharing-icon
-    background-image embedurl('../assets/icons/ui/icon-share.svg')
+    background-image embedurl('../assets/icons/ui/share.svg')
```
Comment (kosssi) [inline src/components/SharingIcon/SharingIcon.styl]: This is on cozy-ui I thinks the path is not good?

Comment (ptbrowne) [reply src/components/SharingIcon/SharingIcon.styl]: Why ? The stylus directory of cozy-ui is added to the stylus path.

+ stylus
+ assets

So you go up one directory and then you're good.

Comment (ptbrowne) [reply src/components/SharingIcon/SharingIcon.styl]: Ca n'a pas fonctionné le changement, je pense qu'avec le ../assets/icons on le force à aller regarder dans cozy-ui.

```
  Module not found: Error: Can't resolve 'icons/ui/share.svg' in '/Users/pbrowne/code/cozy/banks/src/components/SharingIcon'
```

File: src/ducks/transactions/TransactionModal.spec.jsx (hunk 1 above)
Comment (ptbrowne) [reply src/ducks/transactions/TransactionModal.spec.jsx]: J'ai enlevé

Comment (ptbrowne) [reply src/ducks/transactions/TransactionModal.spec.jsx]: Yes

**Pull request merged**